ambry-tools/src/integration-test/java/com.github.ambry.clustermap/HelixBootstrapUpgradeToolTest.java

```diff
@@ -135,11 +135,12 @@ public void testEverything() throws Exception {
     testHardwareLayout.addNewDataNodes(numNewNodes);
     testPartitionLayout.addNewPartitions(numNewPartitions);
 
-    // Next, mark all previous partitions as READ_ONLY
+    // Next, mark all previous partitions as READ_ONLY, and change their replica capacities.
     for (PartitionId partitionId : testPartitionLayout.getPartitionLayout().getPartitions()) {
       if (partitionIdsBeforeAddition.contains(((Partition) partitionId).getId())) {
         Partition partition = (Partition) partitionId;
         partition.partitionState = PartitionState.READ_ONLY;
+        partition.replicaCapacityInBytes += 1;
```

Comment:
This is testing the case where replica capacity changes between upgrades, so adding 1 to change it.